app/resources/assets/sass/app.scss

```diff
@@ -28,6 +28,8 @@
 @import "_sign_in.scss";
 @import "_sign_up.scss";
 @import "_social_media_buttons.scss";
+@import "_suggestion_list.scss";
```

Comment:
I like that you followed the way the code was organized here by making files specific to each page.